tensor.go

```diff
@@ -28,69 +28,69 @@ const (
 	TfLiteInt8
 )
 
-// TfLiteTensor represents TensorFlow Lite Tensor.
-type TfLiteTensor struct {
+// Tensor represents TensorFlow Lite Tensor.
+type Tensor struct {
 	tensor *C.TfLiteTensor
```

Comment:
It might be a good idea to rename this to data

Reply:
Same situation that TFLiteModel, for compatibility with the name into C Library, but seems more intuitive and simple use only Tensor.

The Tensor is the data structure, it is a relevant concept into the TensorFlow ecosystem, should be maintained but we can explore ideas into the forum.

Reply:
Have you consider composing the `C.TfLiteTensor` inside the `Tensor` data structure? That's something widely used in Go:

Instead of having this:
```go
type Tensor struct {
	tensor *C.TfLiteTensor
}
```

We would have something like this:

```go
type Tensor struct {
	*C.TfLiteTensor
}
```

But some testing is required in order to make sure this works.

Reply:
`Tensor` will have all fields inside `TfLiteTensor` with the second approach.

Reply:
I like the second approach, we can prepare this modification on a future PR. could be a interesting approach. Let me test the behavior.

Reply:
Sure thing, will reflect this change later this week.